recipes/gtkmm/2.x.x/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "2.24.5":
+    sha256: 0680a53b7bf90b4e4bf444d1d89e6df41c777e0bacc96e9c09fc4dd2f5fe6b72
+    url: https://download.gnome.org/sources/gtkmm/2.24/gtkmm-2.24.5.tar.xz
```

Comment:
Does this actually co-operate with the current cci `gtk/system` recipe?
We don't have a gtk recipe built from sources (yet).